src/app/hero/home/home-container/home-container.component.html

```diff
@@ -1,2 +1,19 @@
 <mu-header></mu-header>
+<div class="content">
+  <div class="funtionalities">
+    <div class="search-bar">
+      <input type="text" placeholder="  Search... " #searchedString>
+      <button type="button" (click)="searchHeroes(searchedString.value)">GO</button>
+    </div>
+    <div class="sort-button" (click)="sortHeroes()">
+      <button type="button">SORT</button>
+    </div>
+  </div>
+  <mu-loader></mu-loader>
+  <div class="card-container">
+    <div class="card" *ngFor="let hero of allHeroes | async">
+      <mu-card [hero]="hero"></mu-card>
```

Comment:
what about being redirected to hero detail page on click?

Reply:
That I am implementing using routerLink in the card.component.html